cmd/containerd/main_windows.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"context"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"google.golang.org/grpc"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/containerd/execution"
+	"github.com/urfave/cli"
+)
+
+func appendPlatformFlags(flags []cli.Flag) []cli.Flag {
+	return flags
+}
+
+func processRuntime(ctx context.Context, runtime string, root string) (execution.Executor, error) {
+	// TODO
+	return nil, nil
```

Comment:
If I do that, it makes it much harder to find the next point of issue... Hence I'd rather leave them no-op'd for now to continue making progress.